src/benchmark_gemm_numerics.py

```diff
@@ -126,6 +126,73 @@ def gemm_fp8_rowwise_calculate_metrics(
     total_flops, total_flops_all_devices = handle_based_on_sharding(total_flops, SHARDING_STRATEGY)
     return unified_flops_metrics(m, n, k, time_ms_list, total_flops, total_flops_all_devices, PEAK_FLOPS_PER_DEVICE)
 
+def gemm_fp8_rowwise_w_dequantize(
+    m: int, k: int, n: int, num_runs: int = 1, trace_dir: str = None
+) -> Dict[str, Any]:
+    """FP8-Rowwise GEMM with dynamic scaling factors."""
+    def f(x, qy):
+        with jax.named_scope(MARKER):
+            qx = qpl.quantize(x, qtype=jnp.float8_e4m3fn, scale_dtype=jnp.float32, calibration_method="absmax", channelwise_axes=[0])
+            acc = jax.numpy.einsum("ij,jk->ik", qx.qvalue, qy.qvalue, preferred_element_type=jnp.float32).astype(jnp.float32)
```

Comment:
For the benchamarks to quantize both inputs on the fly and with dequant:

add 
```
qy = qpl.quantize(y, qtype=jnp.float8_e4m3fn, scale_dtype=jnp.float32, calibration_method="absmax", channelwise_axes=[1])
```
after qx=....